### PR TITLE
Make changes for iteration 3 spec harness

### PR DESCRIPTION
**Pull request by lgsriclas**

### CODE REVIEW

**Review by michab17 — APPROVED** [review]

File: lib/invoice_item_repository.rb [hunk 1]
```diff
@@ -2,24 +2,25 @@
...
   end
 
   def inspect
     "#<#{self.class} #{@invoice_items.size} rows>"
   end
 
-  def all
-    @rows.map do |row|
+  def generate(path)
+    rows = CSV.read(@path, headers: true, header_converters: :symbol)
```
Comment (samueldevine) [inline lib/invoice_item_repository.rb]: we can update the `@path` to `path` here

File: lib/invoice_item_repository.rb [hunk 2]
```diff
@@ -2,24 +2,25 @@
...
       InvoiceItem.new(row)
     end
   end
 
   def find_by_id(id)
-    result = @all.find do |row|
+    @all.find do |row|
       row.id == id
     end
     result
```
Comment (samueldevine) [inline lib/invoice_item_repository.rb]: no need for this result anymore :)

**Review by samueldevine — CHANGES_REQUESTED** [review]
very minor changes added - great refactor!

**Review by samueldevine — APPROVED** [review]

**Comment by samueldevine** [comment]
looks good to me! ready to merge

**Pull request merged**